00XX-efficient-msms-for-non-native-pickles-verification.md

```diff
@@ -473,9 +462,7 @@ Think about the lessons from other blockchain projects or similar updates and pr
 
 ## Unresolved questions
 
-1. Engineering problem: how exactly does the accumulation trick with the additive lookup tables work?
-   - If we go with the circuit layout that fits $2^{15}$ additions into $2^{15}$ rows having exactly one addition per row, we need to make sure the total joint accumulator (between folding iterations) is being updated properly. This requires altering the folding protocol checks.
-2. When splitting MSM into chunks --- is it not problematic that we have to enforce the right order of the chunks? The soundness of the approach relies heavily on the impossibility to swap these chunks around.
+1. When splitting MSM into chunks --- is it not problematic that we have to enforce the right order of the chunks? The soundness of the approach relies heavily on the impossibility to swap these chunks around.
```

Comment:
No.

> The soundness of the approach relies heavily on the impossibility to swap these chunks around.

I don't think this is true/important at the stage we're describing in the RFC. In particular, we have a fixed number `N` of iterations, and for each `0 <= i < N-1` instance `i` has output that's consistent with instance `i+1`'s input.

Suggest deleting.

Reply:
At this stage it's not important, agreed, so will remove :+1: 

Longer-term, when we add folding and IVCs... I would need to understand this more to make sure it's properly sound.